meson.build

```diff
@@ -215,6 +215,7 @@ if not disable_crypt_option
 endif
 
 if not disable_bsd_option
+	rtdl_include_dirs += include_directories('options/bsd/include')
```

Comment:
Why does ``rtdl`` need the BSD includes?

Reply:
I don't recall the exact reason (but there was one), but I suspect it is `unistd.h` pulling in `bits/bsd/bsd_unistd.h` if the BSD option is enabled.